### PR TITLE
[Docs] Remove Reference to `to_chart` in OBBject.

### PR DESCRIPTION
**Pull request by deeleeramone**
1. **Why**? (1-3 sentences or a bullet point list):

    - `to_chart()` is no longer a property of OBBject.

2. **What**? (1-3 sentences or a bullet point list):

     - Removes the references in the "architectural considerations" page to `to_chart()`. 

3. **Impact** (1-2 sentences or a bullet point list):
     - Docs only.

### CODE REVIEW

File: website/content/platform/development/developer-guidelines/architectural_considerations.md [hunk 1]
```diff
@@ -97,12 +97,10 @@ The `Fetcher` class implementation is based on the [TET pattern](/platform/devel
...
 
-The class includes several fields such as results, provider, warnings, chart, and extra, which respectively represent the fetched data, the data provider, any warnings generated during data fetching, an optional chart object for visualizing the data, and a dictionary for any additional information.
+The class includes several fields such as results, provider, warnings, and extra, which respectively represent the fetched data, the data provider, any warnings generated during data fetching, and a dictionary for any additional information.
 
 The class provides several methods for converting the fetched data into different formats, including `to_df` (or `to_dataframe`) for converting to a pandas DataFrame, `to_polars` for converting to a Polars DataFrame, `to_numpy` for converting to a numpy array, and `to_dict` for converting to a dictionary.
 
-The `to_chart` method allows for creating or updating a chart based on the fetched data, and the show method is used to display the chart.
-
 The class also includes a `__repr__` method for a human-readable representation of the object, and a `model_parametrized_name method` for returning the model name with its parameters.
```
Comment (deeleeramone) [inline website/content/platform/development/developer-guidelines/architectural_considerations.md:104]: @hjoaquim, when I run, `data.model_parameterized_name()`, I get:

```
TypeError: OBBject.model_parametrized_name() missing 1 required positional argument: 'params'
```

File: website/content/platform/development/developer-guidelines/architectural_considerations.md [hunk 2]
```diff
@@ -97,12 +97,10 @@ The `Fetcher` class implementation is based on the [TET pattern](/platform/devel
 
 The OBBject class is a generic class in the OpenBB platform that represents a standardized object for handling and manipulating data fetched from various providers. It extends the `Tagged` class and uses Python's generics to allow flexibility in the type of results it can handle.
 
-The class includes several fields such as results, provider, warnings, chart, and extra, which respectively represent the fetched data, the data provider, any warnings generated during data fetching, an optional chart object for visualizing the data, and a dictionary for any additional information.
```
Comment (hjoaquim) [inline website/content/platform/development/developer-guidelines/architectural_considerations.md:100]: Why removing the mention to `chart`? It is still a field of the `OBBject`

File: website/content/platform/development/developer-guidelines/architectural_considerations.md (hunk 1 above)
Comment (hjoaquim) [reply website/content/platform/development/developer-guidelines/architectural_considerations.md:104]: Indeed. Not sure what the intention was here tbh, but we can either remove it or change it to `schema()` instead. Wdyt?

**Review by hjoaquim — CHANGES_REQUESTED** [review]

File: website/content/platform/development/developer-guidelines/architectural_considerations.md (hunk 2 above)
Comment (deeleeramone) [reply website/content/platform/development/developer-guidelines/architectural_considerations.md:100]: Ooooops! My bad, I got carried away with my yeeting by the looks of it!

**Review by hjoaquim — APPROVED** [review]

**Pull request merged**